app/src/main/java/com/sunlightlabs/congress/services/LegislatorService.java

```diff
@@ -167,6 +185,15 @@ else if (!json.isNull("id"))
                 legislator.phone = role.getString("phone");
         }
 
+        // most minimal form: list of cosponsors, ID and name only
+        else if (!json.isNull("cosponsor_id")) {
+            legislator.bioguide_id = json.getString("cosponsor_id");
+            String[] names = Legislator.splitName(json.getString("name"));
+            legislator.first_name = names[0];
+            legislator.last_name = names[1];
+            legislator.cosponsored_on = ProPublica.parseDateOnly(json.getString("date"));
+        }
```

Comment:
This is a minimal form of legislator data, shown on `cosponsors` results. Tracking expanding the data available there at https://github.com/propublica/congress-api-docs/issues/69